bluemira/equilibria/coils/_grouping.py

```diff
@@ -955,6 +964,11 @@ def __init__(
     ):
         if len(coils) == 1:
             coils = (coils[0], deepcopy(coils[0]))
+            if "U" in coils[0].name:
+                coils[1].name = coils[0].name.replace("U", "L")
```

Comment:
This is dangerous eg a coil named 'ABUC_1' would be renamed to 'ABLC_1' which is not the aim